akka-docs/src/test/scala/docs/persistence/PersistencePluginDocSpec.scala

```diff
@@ -47,6 +47,16 @@ object PersistencePluginDocSpec {
       //#native-config
       akka.persistence.journal.leveldb.native = off
       //#native-config
+
+      //#compaction-intervals-config
+      # Number of deleted messages per persistence id that will trigger journal compaction
+      akka.persistence.journal.leveldb.compaction-intervals {
+        # persistence-id-1 = interval-1
```

Comment:
might be better with real values here instead of comments
```
persistence-id-1 = 30s
persistence-id-2 = 60s
```

Reply:
Actually the intervals are message intervals not time ones, which actually proves the point of putting some actual values to the documentation. Will fix